dev/devicelab/manifest.yaml

```diff
@@ -338,6 +338,14 @@ tasks:
     required_agent_capabilities: ["linux/android"]
     flaky: true
 
+  web_size__compile_test:
+    description: >
+      Measures the size of a dart2js bundle.
+    stage: devicelab
+    required_agent_capabilities: ["linux/android"]
+    flaky: true
```

Comment:
Curious: what does flaky:true mean?

Reply:
(sorry clicked on "resolve" by accident)

It is a common practice to mark a new devicelab test as flaky until you are sure that it passes and is stable. Otherwise, you risk breaking the build for everyone.